cumulusci/tasks/bulkdata.py

```diff
@@ -266,14 +266,16 @@ def _run_task(self):
             self.logger.info("Running Job: {}".format(name))
             result = self._load_mapping(mapping)
             if result != "Completed":
-                break
+                raise BulkDataException(
+                    "Job {} did not complete successfully".format(name)
+                )
```

Comment:
When there's a failed batch we should probably make _parse_job_state grab the `stateMessage` and pass it along so we can show a more informative message here.